torchaudio/models/decoder/_ctc_decoder.py

```diff
@@ -113,6 +114,83 @@ class CTCHypothesis(NamedTuple):
     timesteps: torch.IntTensor
 
 
+class CTCDecoderLMState(_LMState):
+    """Language model state.
+
+    :ivar Dict[int] children: Map of indices to LM states
+    """
+
+    def child(self, usr_index):
+        """Returns child corresponding to usr_index, or creates and returns a new state if input index
+        is not found.
+
+        Args:
+            usr_index (int): index corresponding to child state
+
+        Returns:
+            CTCDecoderLMState: child state corresponding to usr_index
+        """
+        return super().child(usr_index)
```

Comment:
@mthrok do you think calling super here is fine? am running into an issue where nothing is returned if I just use pass